src/km.js

```diff
@@ -299,7 +299,10 @@
     ZMI() { D.prf.zoom(Math.min(12, D.prf.zoom() + 1)); },
     ZMO() { D.prf.zoom(Math.max(-10, D.prf.zoom() - 1)); },
     ZMR() { D.prf.zoom(0); },
-
+    RLB() {
+      D.prf.lbarOrder(D.lb.order);
+      D.ide.lbarRecreate();
```

Comment:
`D.ide.lbarRecreate()` ought to be triggered automatically as it is set as a listener for a change to lbarOrder. Have you tried removing this line?

Reply:
Right on! If I read correctly, we add a listener for event [here](https://github.com/Lindeneg/ride/blob/reset-lb-layout-action/src/ide.js#L249) that indeed calls lbarRecreate. So, I'll remove the line from `RLB`.